tests/LongRunning/TranslationRecoReconnectTests.ts

```diff
@@ -215,50 +213,32 @@ test("Test new connection on empty push stream for translator", (done: jest.Done
     s = sdk.SpeechTranslationConfig.fromSubscription(Settings.SpeechSubscriptionKey, Settings.SpeechRegion);
     s.speechRecognitionLanguage = "en-US";
     s.addTargetLanguage("de-DE");
-    const startTime: number = Date.now();
-    let longPauseOccured: boolean = false;
-    let shortPause: boolean = false;
+    let disconnected: boolean = false;
+    let reconnected: boolean = false;
+    let reconnectTime: number;
 
     const openPushStream = (): sdk.PushAudioInputStream => {
         // create the push stream we need for the speech sdk.
         const pushStream: sdk.PushAudioInputStream = sdk.AudioInputStream.createPushStream();
-        const chunkSize: number = 1024;
+        const chunkSize: number = 512;
```

Comment:
curiosity thing: why reduce the chunk size?

Reply:
slows down the intake (and thus the sending of "audio" messages), to stretch out the session to 45 minutes to ensure the connection stays up long enough to disconnect and then have enough audio left over for one recognition post-reconnect.